_1327/documents/templates/documents_edit.html

```diff
@@ -403,6 +403,26 @@ <h4 class="modal-title" id="deleteAutosaveHeader">{% trans "Do you really want t
 		// add datepicker to date inputs
 		$("input[name*='date']:not([readonly='True'])").datepicker( $.datepicker.regional["{{ LANGUAGE_CODE }}"] );
 
+		function getCurrentTime() {
+			const currentDate = new Date();
+			const currentHour = currentDate.getHours().toString();
+			let currentMinute = currentDate.getMinutes().toString();
+			if(currentMinute.length == 1) currentMinute = "0" + currentMinute;
```

Comment:
please add a leading zero for the hours as well. i would have preferred proper string formatting, but it seems like javascript is not really good with that.